### PR TITLE
Feature/display feedback

### PR DESCRIPTION
**Pull request by knees4bees**
### What does this PR do?
* Creates basic version of `StudentFeedback` component with very lightweight styling, basic skeleton for dynamic access of category data, and import of `scss` file to access Turing color variables.

### Anything specific the reviewer should look at?
* Take a look at the files; see if anything looks wonky. I may not be following best practices in terms of data storage and retrieval and file import.

Note that the score for each category is just a number in a box right now, but my idea for it is to make it a bar with length according to the score. I think I have a good resource for this! Just ran out of time to implement it.

### CODE REVIEW

File: src/components/StudentFeedback.vue [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+  data () {
+    return {
+      categories: ['JavaScript', 'React', 'Professionalism']
+    }
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+  @import '@/styles/styles.scss';
```
Comment (m1073496) [inline src/components/StudentFeedback.vue:54]: Does this need to go here if it is also being imported in main.js?

**Review by m1073496 — COMMENTED** [review]
Hey @knees4bees, I fetched this branch and checked out the UI and it looks good! Nice work! I left one comment, which is really just a general question since I'm unsure about how Vue requires things to be imported. As to your questions on the PR, I think @piknikki would be better suited to answering those. Thanks!

File: src/components/StudentFeedback.vue (hunk 1 above)
Comment (piknikki) [reply src/components/StudentFeedback.vue:54]: I don't think so. You have access to it through the main.js import.

Comment (piknikki) [reply src/components/StudentFeedback.vue:54]: And a solid start! I wonder if we can make those category renders more dynamic? Categories is an array, so we should be able to use a v-for directive to iterate through them. Maybe we can make that an array of objects like the way they'll come in from the api, with a category, a score, and the category feedback (which I love, btw!!).

**Review by piknikki — COMMENTED** [review]
A great start!!  Should we merge it and then keep working on a new branch?  I've done some work we can talk through today, in moving the links to module, that may affect your work. But I think we should talk through it.

File: src/components/StudentFeedback.vue (hunk 1 above)
Comment (m1073496) [reply src/components/StudentFeedback.vue:54]: I ended up needing to import here as well, so it looks like the main.js file import doesn't carry over into the components.

Comment (knees4bees) [reply src/components/StudentFeedback.vue:54]: Yes, let's absolutely make this more dynamic!!

**Pull request merged**